mithril.js

```diff
@@ -28,6 +28,19 @@
 		return "v0.2.5"
 	}
 
+	var div = document.createElement("div")
+	var frag = document.createDocumentFragment()
+
+	function toDOM(HTML){
+		div.innerHTML = HTML
```

Comment:
@isiahmeadows EXACTLY. The fragment is emptied when it is appended. There is no desire to create a new allocation: the same fragment can be used again and again, in the same way the div element is.